### PR TITLE
create tests for b2c-sales-units

### PR DESCRIPTION
**Pull request by Mb-Hala**

### CODE REVIEW

File: tests/api/b2c/glue/weight_unit_endpoints/sales_units/negative.robot [hunk 1]
```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/sales_units/negative.robot:5]: please add this before tests
```
ENABLER
    TestSetup
```

and this in the setup

```
Test Setup     TestSetup
Default Tags    glue
```

File: tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot [hunk 2]
```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot:5]: please add this before tests

ENABLER
    TestSetup

and this in the setup

```
Test Setup     TestSetup
Default Tags    glue
```

File: tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Get_sales_units_for_product_without_sales_units
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot:7]: tests fail when run like that even after the changes requested above - please fix this
robot -v env:api_b2c -d Results tests/api/b2c/glue/weight_unit_endpoints/sales_units on https://glue.de.b2c.internal-testing.demo-spryker.com

image

**Comment by vadym-uskov** [comment]
N/A for b2c and mp_b2c

**Pull request closed without merging**